lib/src/style/QlementineStyle.cpp

```diff
@@ -1458,7 +1458,7 @@ void QlementineStyle::drawControl(ControlElement ce, const QStyleOption* opt, QP
 
         // Little hack to avoid having a checked extension button.
         auto buttonState = optToolButton->state;
-        const auto isExtensionButton = w->objectName() == QStringLiteral("qt_toolbar_ext_button");
+        const auto isExtensionButton = (w && w->objectName() == QStringLiteral("qt_toolbar_ext_button"));
```

Comment:
We could actually `return` if there is no widget at all!

Reply:
After discussion we decided to not use 'return' to handle specific case when widget is not specified.